test/rules/typedef-whitespace/both/onespace/test.ts.lint

```diff
@@ -220,7 +220,7 @@ function withPreceedingSpaceNoSubsequentSpaceFn(a :number, b :number) :number {
     }
 }
 
-function withPreceedingSpaceWithSubsequentSpaceFn(a : number, b : number) : number {
+function withPreceedingSpaceWithSubsequentSpaceFn(a : number, b ?: number) : number {
```

Comment:
I'll ask around and see if `varName ?: type` is used elsewhere in the TS community and considered good practice. If it's decently used, we can proceed with this PR. If not, you can always compile this rule yourself and use it as a custom rule.

Worth noting though that the original commenter in #811 wanted the space to come after the `?`, not before.